linetools/lists/linelist.py

```diff
@@ -369,19 +389,22 @@ def set_extra_columns_to_datatable(self, abundance_type='solar', ion_correction=
 
         """
         # Avoid redo (especially for caching)
-        if ('ion_name' in self._data.keys()) and (not redo):
+        if ('ion_name' in self._extra_table.keys()) and (redo is False):
             return
 
         if self.list not in ['HI', 'ISM', 'EUV', 'Strong', 'H2']:
             warnings.warn('Not implemented: will not set relative strength for LineList: {}.'.format(self.list))
             return
 
         # Set ion_name column
+        ion_name = np.array([' '*20]*len(self.name)).astype(str)
```

Comment:
well, I'm not sure we have a convention at all.  
Only 2 different prescriptions to generate them.

It might be good to have one method that does them.
But I think we are ok for now.